yarn.lock

```diff
@@ -8503,7 +8491,7 @@ jquery-deferred@^0.3.0:
   resolved "https://registry.yarnpkg.com/jquery-deferred/-/jquery-deferred-0.3.1.tgz#596eca1caaff54f61b110962b23cafea74c35355"
   integrity sha1-WW7KHKr/VPYbEQlisjyv6nTDU1U=
 
-jquery@^3.3.1, jquery@^3.4.1:
+jquery@^3.3.1:
```

Comment:
hmm, looks like some other dep is requiring this? `yarn why jQuery reveals nothing. :-\

Reply:
because `yarn why` is case sensitive. whoops.

ember-source depends on it :-\
must be a dev-time thing?

Reply:
It's `ember-source@~3.4.0` https://github.com/embermap/ember-data-storefront/blob/029b71b9ab46d7163f2f338139065a400aa785e3/yarn.lock#L6059-L6077